configure.ac

```diff
@@ -387,29 +387,31 @@ SECP_CFLAGS="$SECP_CFLAGS $WERROR_CFLAGS"
 ### Handle module options
 ###
 
-if test x"$enable_module_ecdh" = x"yes"; then
-  SECP_CONFIG_DEFINES="$SECP_CONFIG_DEFINES -DENABLE_MODULE_ECDH=1"
-fi
```

Comment:
Why change the order in configure.ac?

Reply:
To have the same order as in CMake, I think this makes maintenance easier. Let me add this rationale to the commit message? Or do you think I should not do this change?

Reply:
No I think it's fine.